estimateTextWidth.test.js

```diff
@@ -0,0 +1,14 @@
+import test from 'ava';
+import estimateTextWidth from './estimateTextWidth';
+
+test('test a few simple strings', t => {
+    t.is(estimateTextWidth('a'), 9);
+    t.is(estimateTextWidth('aa'), 18);
+    t.is(estimateTextWidth('aaa'), 27);
+});
+
+test('test some more strings', t => {
+    t.snapshot(estimateTextWidth('hello world'));
+    t.snapshot(estimateTextWidth('1234.56'));
+    t.snapshot(estimateTextWidth('1234.56 $'));
```

Comment:
I this case I would use `t.is` since you only multiply integers like you did in the test above. Saves the 2 snapshot files and is faster. 

`t.snapshot` is generally a good idea if you want to compare big objects or DOM nodes. Things like that.

Reply:
ah good to know